lib/simulator-xcode-6.js

```diff
@@ -617,6 +617,25 @@ class SimulatorXcode6 extends EventEmitter {
     return await settings.updateLocationSettings(this, bundleId, authorized);
   }
 
+  /**
+   * Enable/Disable reduce motion.
+   *
+   * @param {string} bundleId - The application ID to update.
+   * @param {boolean} reduceMotion - Whether or not to enable it.
+   */
+  async reduceMotion (bundleID, reduceMotion = true) {
+    if (await this.isRunning()) {
+      await this.endSimulatorDaemon();
+    } else if (await this.isFresh()) {
+      await this.launchAndQuit(false, STARTUP_TIMEOUT);
+    }
+    await settings.reduceMotion(this, bundleID, reduceMotion);
+  }
+
+  async readSettings (plist) {
```

Comment:
what is the purpose of `readSettings` method?

Reply:
I was using it for functional testing, let me see if I can have a different approach besides exposing this method